### PR TITLE
Add configurability to executing nodes.

### PR DESCRIPTION
**Pull request by Carreau**
A bit hacky but should help with #12151

### CODE REVIEW

**Comment by Carreau** [comment]
cc @alexmojaki

File: IPython/core/ultratb.py [hunk 1]
```diff
@@ -716,7 +716,7 @@ def get_records(self, etb, number_of_lines_of_context, tb_offset):
         before = context - after
         if self.has_colors:
             style = get_style_by_name('default')
-            style = stack_data.style_with_executing_node(style, 'bg:#00005f')
+            style = stack_data.style_with_executing_node(style, self.color_scheme_table.active_colors.ExecutingNode)
```
Comment (alexmojaki) [inline IPython/core/ultratb.py]:
```suggestion
            style = stack_data.style_with_executing_node(style, self.Colors.ExecutingNode)
```

File: IPython/core/excolors.py [hunk 2]
```diff
@@ -96,7 +96,8 @@ def exception_colors():
         excName = C.LightRed,
         line = C.Yellow,
         caret = C.White,
-        Normal = C.Normal
+        Normal = C.Normal,
+        ExecutingNode = 'bg:#00005f'
```
Comment (alexmojaki) [inline IPython/core/excolors.py]:
```suggestion
        ExecutingNode = 'bg:#00005f',
```

File: IPython/core/excolors.py [hunk 3]
```diff
@@ -126,6 +127,7 @@ def exception_colors():
         line = C.Red,
         caret = C.Normal,
         Normal = C.Normal,
+        ExecutingNode = 'bg:#005f00'
```
Comment (alexmojaki) [inline IPython/core/excolors.py]:
```suggestion
        ExecutingNode = 'bg:#005f00',
```

Keep diffs minimal :)

File: IPython/core/excolors.py [hunk 4]
```diff
@@ -154,6 +156,7 @@ def exception_colors():
         line = C.Red,
         caret = C.Normal,
         Normal = C.Normal,
+        ExecutingNode = 'bg:#dddddd'
```
Comment (alexmojaki) [inline IPython/core/excolors.py]:
```suggestion
        ExecutingNode = 'bg:#dddddd',
```

**Review by alexmojaki — CHANGES_REQUESTED** [review]

**Comment by alexmojaki** [comment]
Seems to me that it would be easier to first have pygments styles sorted for each option and then we could see what the executing colors look like behind syntax highlighted code.

**Comment by Carreau** [comment]
> Seems to me that it would be easier to first have pygments styles sorted for each option and then we could see what the executing colors look like behind syntax highlighted code.

Yes the whole color think is a mess. I was just hoping to use this as a workaround and slowly progress toward a clean solution.

File: IPython/core/excolors.py [hunk 5]
```diff
@@ -69,6 +69,7 @@ def exception_colors():
         line = C.NoColor,
         caret = C.NoColor,
         Normal = C.NoColor
```
Comment (Carreau) [inline IPython/core/excolors.py:71]:
```suggestion
        Normal = C.NoColor,
```

**Pull request closed without merging**